keywords.txt

```diff
@@ -345,6 +341,14 @@ MY_ESP8266_SSID	LITERAL1
 # MY_RF69_RESET
 # MY_RF69_SPI_CS
 # MY_TRANSPORT_DONT_CARE_MODE
+# MY_ESP8266_BSSID	LITERAL1
+# MY_ESP8266_HOSTNAME	LITERAL1
+# MY_ESP8266_PASSWORD	LITERAL1
+# MY_ESP8266_SSID	LITERAL1
+# MY_ESP32_SSID	LITERAL1
```

Comment:
Maybe we don't need to blacklist the esp32 keywords since they were never released?